xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecAndroidMediaCodec.cpp

```diff
@@ -399,16 +399,30 @@ bool CDVDAudioCodecAndroidMediaCodec::AddData(const DemuxPacket &packet)
         return false;
 
       CJNIMediaCodecCryptoInfo *cryptoInfo(0);
+      CJNIMediaCodecCryptoInfoPattern* cryptoInfoPattern(nullptr);
       if (!!m_crypto->get_raw() && packet.cryptoInfo)
       {
+        if (CJNIBase::GetSDKVersion() < 25 &&
+            packet.cryptoInfo->mode == CJNIMediaCodec::CRYPTO_MODE_AES_CBC)
+        {
+          CLog::Log(LOGERROR, "CDVDAudioCodecAndroidMediaCodec::AddData: Device API does not "
```

Comment:
```suggestion
          CLog::LogF(LOGERROR, "Device API does not support CBCS decryption");
```
With LogF the class name and method are automatically added